rsbuild.config.ts

```diff
@@ -0,0 +1,144 @@
+import { defineConfig, loadEnv } from '@rsbuild/core';
+import { pluginEslint } from '@rsbuild/plugin-eslint';
+import { pluginNodePolyfill } from '@rsbuild/plugin-node-polyfill';
+import { pluginReact } from '@rsbuild/plugin-react';
+import { pluginSass } from '@rsbuild/plugin-sass';
+import { pluginSvgr } from '@rsbuild/plugin-svgr';
+
+const { publicVars, rawPublicVars } = loadEnv({ prefixes: ['REACT_APP_'] });
+
+export default defineConfig({
+  plugins: [
+    pluginReact(),
+    pluginSvgr({ mixedImport: true }),
+    pluginNodePolyfill(),
+    pluginSass(),
+    pluginEslint({
+      enable: process.env.NODE_ENV === 'development',
+      eslintPluginOptions: {
+        cwd: __dirname,
+        configType: 'flat'
+      }
+    })
+  ],
+  server: {
+    port: 8000
+  },
+  tools: {
+    // TODO: See if still needed
```

Comment:
Will we need these commented code in the future?

Reply:
It's mainly kept as reference since the original file was deleted (so it would be hard to get a "file history" since the old webpack config won't be present in the current repo anymore).

Now that this PR is merged, it's free to be deleted as one can always look at the file history of rsbuild.config.ts to easily see what the original webpack config was